src/cpp/session/modules/SessionThemes.R

```diff
@@ -783,6 +783,40 @@
 })
 
 # Worker Functions =================================================================================
+# Gets the install location.
+#
+# @param global    Whether to get the global or local install dir.
+#
+# Returns the install location.
+.rs.addFunction("getThemeInstallDir", getThemeInstallDir <- function(global) 
+{
+   # Copy the file to the correct location.
+   installLocation <- ""
+   if (global)
+   {
+      installLocation <- Sys.getenv("RS_THEME_GLOBAL_HOME", unset = NA)
+      if (is.na(installLocation))
+      {
+         if (grepl("windows", Sys.info()[["sysname"]], ignore.case = TRUE))
+         {
+            installLocation <- file.path("%ProgramData%", "RStudio", "themes")
```

Comment:
Is this always going to match `core::system::systemSettingsPath`? (If not it may be necessary to `.Call` to retrieve the settings path.)

Reply:
%programdata% is an environment variable, does that get expanded somewhere? The expansion of %programdata% will be correct (i.e. match `systemSettingsPath`).

Reply:
Ah, for some reason I thought I had this working on windows, but I'm testing again and it doesn't appear to work. I can change it to `Sys.getenv("ProgramData")` instead.